### PR TITLE
dict .keys() and .values()

### PR DESCRIPTION
**Pull request by jayvdb**
Related to https://github.com/adsharma/py2many/issues/248

### CODE REVIEW

File: pyjl/transpiler.py [hunk 1]
```diff
@@ -231,6 +231,21 @@ def visit_Bytes(self, node):
         bytes_str = "{0}".format(node.s)
         return bytes_str.replace("'", '"')  # replace single quote with double quote
 
+    def visit_Compare(self, node):
+        left = self.visit(node.left)
+        right = self.visit(node.comparators[0])
+
+        comparator_typename = self._typename_from_annotation(node.comparators[0])
+        if comparator_typename and comparator_typename.startswith("Dict"):
```
Comment (jayvdb) [inline pyjl/transpiler.py:239]: im not keen on these string comparisions.
I see `hasattr(node, "container_type")` elsewhere .. maybe that will work here.

Comment (adsharma) [reply pyjl/transpiler.py:239]: yeah - `container_type` is much more robust. Let's go with that.

Comment (adsharma) [reply pyjl/transpiler.py:239]: `*generic_container_type` if you're comparing to `Dict`

Comment (jayvdb) [reply pyjl/transpiler.py]: Done. ping @adsharma

**Pull request merged**